README.md

```diff
@@ -1,20 +1,20 @@
-[![Build Status](https://travis-ci.org/WebAssembly/sexpr-wasm-prototype.svg?branch=master)](https://travis-ci.org/WebAssembly/sexpr-wasm-prototype)
+[![Build Status](https://travis-ci.org/WebAssembly/wast2wasm-prototype.svg?branch=master)](https://travis-ci.org/WebAssembly/wast2wasm-prototype)
 
-# sexpr-wasm-prototype
```

Comment:
You probably want to revert the changes to `sexpr-wasm-prototype`, which is the current repo name.